### ISSUE TITLE
[Item] Bryntroll, the Bone Arbiter

### ISSUE THREAD

**Issue by VeilaMidn** [dalaranwow/dalaran-wow#4785]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The procrate of this weapon seems to be too low.
**Expected behaviour**: Tell us what should happen instead
The weapon should proc more frequently but I do not know the exact value. I was reading around on wowhead as well as their patchnotes. Initially the weapon was bugged on retail servers having a too high procrate, but even here it's about halved. I can barely get more than 1.8% of my overall damage from this weapon. Excluding RNG, expected damagevalue is 3-4% overall. The number of procs is also way too low. At a 2 ppm expected procrate, giving an 11% procrate.  Currently it's significantly lower than this. Additionally, this effect should also be a spellhit. I've checked recount and have never seen it miss or resist. This is also incorrect. It should have a basechance to resist as well. I also believe the weapon does not correctly proc from specials, including dot effects (Crusader Strike, Righteous Vengeance) as I had the same amount of procs both from just white hitting a target dummy for about 250 swings and after I had used specials throughout the span of 250 white swings.
**Steps to reproduce the problem**:

1.  Equip Bryntroll, the Bone Arbiter and go hit a target dummy
2.  Watch recount, both with only white hits and using specials.
3.  Procrate is way lower than expected

**Include proofs for this behaviour**
http://www.wowhead.com/item=50415/bryntroll-the-bone-arbiter#comments
A comment made by "axhawk" shows how this player tested this weapon extensively. I do not know how else to provide proof other than this weapon was estimated to be third BiS, after Glorenzelg and Shadowmourne. Currently it barely does more than Hellion Glaive for me, which is significantly worse. The procrate is where a lot of the weapons power comes from, and currently it's not providing that much.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Thank you for the report @VeilaMidn 

_Golden comment from "axhawk"_

> The theorycrafting on this potent weapon is still rather sparse, so I spent several hours testing the limits of the proc and it's dps contribution. First off, the basics:
> The 264 version proc has an average damage of 2250, while the 277 version has an average of 2538.
> The damage does not scale with any character stats (though the dps does, more on that below).
> The damage does scale with any percentage damage increases that affect spell damage, such as Avenging Wrath, Curse of Elements, Blood Presence, etc.
> The drain can proc off of any weapon attack, as well as some odd events such as applying or refreshing effects such as deathknight diseases or the paladin talent Righteous Vengeance. These odd events do have some requirements though. The drain can proc off of the refreshing of some DoTs or debuffs so long as the character is within melee range of and facing the target for the drain. Specific to the deathknight class, the drain can proc off of the disease application of Icy Touch (assuming you are in melee range), can double-proc off Plague Strike (the strike and the disease application), and on each target that is in melee range and in front of you from Pestilence.
> The drain has a base proc rate of 2 procs-per-minute, or 11.3333% chance per swing, and has no internal cooldown.
> The drain can proc multiple times per attack, provided there's are more than one weapon event for the attack. For example, the drain can multi-proc off the following:
> - On any or all of the multiple targets of Cleave, Whirlwind, Divine Storm, Heart Strike, or Sweeping Strikes.
> - From a melee (or special) and it's subsequent seal proc, Blood-Caked Blade proc, or Windfury proc.
> - From both the physical and shadow portions of Scourge Strike.
> The drain can not proc from any spells, excluding the odd events mentioned above.
> The drain is incapable of critting and functions on the spell hit table. However, it doesn't seem to be resistible.
> Now, with a 2 PPM baseline, the weapon will deal 2 procs a minute before any waste or special attacks or talents are factored in, giving it a minimum dps of 75. In practice, the proc will usually provide between 200 and 400 dps, depending on the character's haste rating, spec and rotation (and therefore number of procable attacks per second), spell hit chance, and +damage buffs present. A good estimate is that it will provide approximately 4-8% of a character's dps. For many classes, this means the proc is worth the equivalent of between 400 and 800 additional attack power, though it's place on the best-in-slot list is strongly dependent class, spec, haste rating, and rotation. In general is can be assumed to be near the top of the list, however, as the proc itself provides a fairly respectable amount of dps while also making it easier for the healers to keep you alive through miscellaneous raid damage.

**Comment by Lothloryen** [comment]
Hello there,

Thank you for your report, this has been already fixed and will be on live with the next maintenance.
